packages/react-ui-components/src/SelectBox_ListPreview/selectBox_ListPreview.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable camelcase, react/jsx-pascal-case */
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+
+const Fragment = props => props.children;
+
+/**
+ * **SelectBox_ListPreview is an internal implementation detail of SelectBox**, meant to improve code quality.
+ *
+ * It is used inside SelectBox as the dropdown part.
```

Comment:
But could also be used in the header, for selected element, right?

Reply:
Nope, there, the ListPreviewElement should be used directly I think.